### ISSUE TITLE
Sitewide - Automatic accessibility testing

### ISSUE THREAD

**Issue by gobyrne** [GCTC-NTGC/TalentCloud#678]
# Description
Lighthouse or similar should be automatically run on pull requests similar to unit tests.

**Comment by xander-miller** [comment]
Resolved by Pull Request #739
